app/models/plan.rb

```diff
@@ -479,6 +479,20 @@ def clone_associations(custom)
 
   private
 
+  # act_as_list updates the position every time that a plan is destroyed.
+  # But we do not want to do that when its issuer is going to be deleted anyway.
+  # Additionally, we cannot do: skip_callback :destroy, :after, :decrement_positions_on_lower_items, if: -> { destroyed_by_association }
+  # because it is an 'after' callback and by the time it is executed, destroyed_by_association is already 'nil'
+  def decrement_positions_on_lower_items
```

Comment:
Perhaps it could be done as in

https://github.com/3scale/porta/blob/d3a59a0164713e681138e808a36f0ec05cf45334/app/models/proxy_rule.rb#L129-L135

Reply:
Agree :)